examples/small-study/README.md

```diff
@@ -0,0 +1,61 @@
+# An example `small-study`
+
+## files
+
+```
+.
+├── README.md
+├── built-addons/       # `web-ext build` output
+├── package-lock.json
+├── package.json
+├── src/
+│   ├── studySetup.js   # mostly declaritive study config
+│   ├── background.js   # main script impelementing addon
+│   ├── icons/
+│   │   ├── LICENSE
+│   │   └── shield-icon.svg
+│   ├── manifest.json
+│   └── privileged/     # npm run studyutils copies `browser.study` api here.
+└── web-ext-config.js
+```
+
+## setup / install
+
+```
+npm install -g web-ext
```

Comment:
Web-ext is usually installed as a global.  I can see if there are Good Reasons or Bad Reasons for that though.   I think people make it global so that web-ext is always in path during interactive development.

Reply:
why this?  Because `web-ext` recommends installing as global:  `https://www.npmjs.com/package/web-ext`.